### PR TITLE
[EDU-3832] feat: add use bucket as origin guide

### PR DESCRIPTION
**Pull request by bru-andrade**
### Changes

- Added new guide to use object as bucket
- Added ref links
- Made small addendum to download objects from bucket guide

### Additional links

[EDU-3000]: https://aziontech.atlassian.net/browse/EDU-3000?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[EDU-3832]: https://aziontech.atlassian.net/browse/EDU-3832?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by GabrielAzion — APPROVED** [review]

File: src/content/docs/en/pages/guides/object-storage/bucket-actions/index.mdx [hunk 1]
```diff
@@ -51,6 +51,21 @@ curl --location 'https://api.azion.com/v4/storage/buckets/<bucket_name>/objects/
 --header 'Authorization: Token [TOKEN VALUE]'
 ```
 
+You should now see the contents of the object in your terminal.
+
+:::note
+To download the object to your device locally, add the cURL options `-O` to use the header-provided object name and `-J` to write the output into a file, as follows:
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/guides/object-storage/bucket-actions/index.mdx:57]:
```suggestion
To download locally the object to your device, add the cURL options `-O` to use the header-provided object name and `-J` to write the output into a file, as follows:
```

File: src/content/docs/en/pages/guides/object-storage/use-bucket-as-origin/index.mdx [hunk 2]
```diff
@@ -0,0 +1,349 @@
...
+8. Wait for the deployment process to finish.
+9. Copy the domain of the application, in the format `http://xxxxxxxxxx.map.azionedge.net/`.
+
+:::note
+The link to the edge application allows you to see it on the browser. However, it takes a certain time to propagate and configure the application in Azion’s edge locations. It may be necessary to wait a few minutes for the URL to be activated and for the application page to be effectively displayed in the browser.
+:::
+
+Now you need to configure a new **Object Storage** origin and determine that your application should retrieve content from the bucket and prefix you created. To do so:
+
+1. [Access RTM](https://manager.azion.com/) > **Edge Application**.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/guides/object-storage/use-bucket-as-origin/index.mdx:157]:
```suggestion
1. [Access RTM](/en/documentation/products/guides/how-to-access-rtm/) > **Edge Application**.
```

File: src/content/docs/pt-br/pages/guias/object-storage/use-bucket-as-origin/index.mdx [hunk 3]
```diff
@@ -0,0 +1,349 @@
...
+8. Aguarde o processo de deploy terminar.
+9. Copie o domínio da application, no formato `http://xxxxxxxxxx.map.azionedge.net/`.
+
+:::note
+O link para sua edge application permite que você veja como ela fica no navegador. No entanto, leva um certo tempo para propagar e configurá-la nas edge locations da Azion. Pode ser necessário aguardar alguns minutos para que a URL seja ativada e para que a página da aplicação seja efetivamente exibida no navegador.
+:::
+
+Agora você precisa configurar uma nova origem tipo **Object Storage** e determinar que sua aplicação deve recuperar o conteúdo do bucket e prefixo que você criou. Para fazer isso:
+
+1. [Acesse o RTM](https://manager.azion.com/) > **Edge Application**.
```
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/guias/object-storage/use-bucket-as-origin/index.mdx:157]:
```suggestion
1. [Acesse o RTM](/pt-br/documentacao/produtos/guias/como-acessar-o-rtm/) > **Edge Application**.
```

File: src/content/docs/pt-br/pages/guias/object-storage/use-bucket-as-origin/index.mdx [hunk 4]
```diff
@@ -0,0 +1,349 @@
...
+title: Como usar um bucket do Azion Object Storage como origem de uma edge application estática
+description: Aprenda como usar um bucket como origem de uma edge application para servir arquivos estáticos.
+meta_tags: object storage, storage, cloud, s3, bucket, objetos, arquivos, armazenamento, file storage, origin, origem
+namespace: documentation_guides_object_storage_origin
+permalink: /documentacao/produtos/guias/usar-bucket-como-origem/
+---
+
+import Button from '~/components/Button.astro'
+
+**Azion Object Storage** permite que você crie buckets e os use como fonte de conteúdo através de [Origins](/pt-br/documentacao/produtos/build/edge-application/origins/#object-storage). Este guia abrange como criar e servir uma aplicação estática retirada de um bucket do Object Storage usando a [API da Azion](https://api.azion.com) e o [Real-Time Manager](https://manager.azion.com).
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/object-storage/use-bucket-as-origin/index.mdx:11]: validate link

Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/object-storage/use-bucket-as-origin/index.mdx:11]:
```suggestion
**Object Storage** da Azion permite que você crie buckets e os use como fonte de conteúdo através de [Origins](/pt-br/documentacao/produtos/build/edge-application/origins/#object-storage). Este guia abrange como criar e servir uma aplicação estática recuperada de um bucket do Object Storage usando a [API da Azion](https://api.azion.com) e o [Real-Time Manager](https://manager.azion.com).
```

File: src/content/docs/pt-br/pages/guias/object-storage/use-bucket-as-origin/index.mdx [hunk 5]
```diff
@@ -0,0 +1,349 @@
...
+
+<Button href="/pt-br/documentacao/produtos/store/object-storage/" text="ir para a referência de Object Storage" variant="secondary" />
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/object-storage/use-bucket-as-origin/index.mdx:13]:
```suggestion
<Button href="/pt-br/documentacao/produtos/store/object-storage/" text="saiba mais sobre Object Storage" variant="secondary" />
```

File: src/content/docs/pt-br/pages/guias/object-storage/use-bucket-as-origin/index.mdx [hunk 6]
```diff
@@ -0,0 +1,349 @@
...
+Agora que seu bucket está populado com arquivos, você pode criar uma nova edge application e um domínio para definir o bucket como a origem do conteúdo e servir esses objetos.
+
+### Via RTM
+
+Para facilitar esse processo, use o template [Edge Application Proxy](/pt-br/documentacao/produtos/guias/edge-application-proxy-template/):
+
+1. Acesse o [Real-Time Manager (RTM)](https://manager.azion.com/).
+    - Se você ainda não tem uma conta, crie uma nova [visitando a página de signup](https://manager.azion.com/signup/).
+2. Na página inicial, selecione a opção **Start with a template**.
+3. Encontre o cartão **Edge Application Proxy** e selecione-o.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/object-storage/use-bucket-as-origin/index.mdx:142]:
```suggestion
3. Encontre o card do **Edge Application Proxy** e selecione-o.
```

File: src/content/docs/pt-br/pages/guias/object-storage/use-bucket-as-origin/index.mdx [hunk 7]
```diff
@@ -0,0 +1,349 @@
...
+4. Clique na aba **Settings** para abrir o formulário de configuração.
+5. Dê à sua application um nome fácil de lembrar.
+6. Em **Origin Address**, digite `httpbin.org`. E
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/object-storage/use-bucket-as-origin/index.mdx:145]:
```suggestion
6. Em **Origin Address**, digite `httpbin.org`.
```

File: src/content/docs/pt-br/pages/guias/object-storage/use-bucket-as-origin/index.mdx [hunk 8]
```diff
@@ -0,0 +1,349 @@
...
+    - Esta origem é provisória e será substituída mais tarde por uma origem de Object Storage.
+7. Em **Route to bypass**, digite `/` para especificar o caminho raiz da application.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/object-storage/use-bucket-as-origin/index.mdx:147]:
```suggestion
7. Em **Route to bypass**, digite `/` para especificar o caminho raiz da aplicação.
```

**Review by MarianaAguilera — APPROVED** [review]

**Pull request merged**